modules/ROOT/attachments/admin-api.yaml

```diff
@@ -809,6 +809,156 @@ paths:
+              {}
+      tags: 
+        - Migrations
+    put: 
+      summary: Create migration
+      operationId: add_migration
+      tags: 
+        - Migrations
+      requestBody: 
+        description: Migration configuration
+        required: true
+        content: 
+          application/json:
+            schema:                 
+              oneOf: 
+              - $ref: "#/components/schemas/inbound_migration"
+              - $ref: "#/components/schemas/outbound_migration"
+              discriminator: 
+                propertyName: migration_type
+                mapping: 
+                  inbound: "#/components/schemas/inbound_migration"
+                  outbound: "#/components/schemas/outbound_migration"
+      responses: 
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:                 
+                oneOf: 
+                - $ref: "#/components/schemas/inbound_migration"
+                - $ref: "#/components/schemas/outbound_migration"
+  /v1/migrations/{id}:
+    get:
+      summary: Get migration state
+      operationId: get_migration
+      tags: 
+        - Migrations
+      parameters: 
+        - name: id
+          in: path 
+          required: true
+          schema: 
+            type: integer
+            format: int32  
+      responses: 
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:                 
+                oneOf: 
+                - $ref: "#/components/schemas/inbound_migration"
+                - $ref: "#/components/schemas/outbound_migration"
+    post: 
+      summary: Execute migration action
+      operationId: execute_migration_action
+      tags: 
+        - Migrations
+      parameters: 
+        - name: id 
+          in: path
+          required: true
+          schema: 
+            type: integer
+            format: int32
+        - name: action
+          in: query
+          required: true
+          description: Action to execute for migration
+          schema: 
+            type: string
+            enum: 
+            - prepare
+            - execute
+            - finish
+            - cancel  
+      responses: 
+        200:
+          description: OK
+          content: {}
+    delete:
+      summary: Delete migration
+      description: Delete specified migration.
+      operationId: delete_migration
+      tags: 
+        - Migrations
+      parameters: 
+        - name: id
+          in: path 
+          required: true
+          schema: 
+            type: integer
+            format: int32 
+      responses: 
+        200:
+          description: OK
+          content: {}    
+  /v1/topics/mount:
+    post:
+      operationId: mount_topics 
+      tags: 
+        - Migrations
+      summary: Mount a topic to a cluster
+      description: Mount a topic in object storage to a cluster according to the specified configuration.
```

Comment:
@bashtanov can't the mount also accept a list?   if not that seems asymmetric

Reply:
Yes it can